drv/ksz8463/src/lib.rs

```diff
@@ -222,13 +223,14 @@ impl Ksz8463 {
     pub fn read_dynamic_mac_table(
         &self,
         addr: u16,
-    ) -> Result<Option<KszMacTableEntry>, Error> {
+    ) -> Result<Option<KszRawMacTableEntry>, Error> {
         assert!(addr < 1024);
         self.write(Register::IACR, 0x1800 | addr)?;
         // Wait for the "not ready" bit to be cleared
         let d_71_64 = loop {
             let d = self.read(Register::IADR1)?;
-            if d & (1 << 15) == 0 {
+            // Check bit 71 to see if the register is ready
```

Comment:
Typo?
```suggestion
            // Check bit 7 to see if the register is ready
```

Reply:
This is actually intentional: the various `IADR*` registers form a 72-bit value, and everything is specified in terms of bits in that value.  I added a comment to clarify!